kernelci/config/test.py

```diff
@@ -302,9 +302,10 @@ def __init__(self, fs_type, boot_protocol='tftp', root_type=None,
         self._arch_dict = {}
         self._ramdisk = ramdisk
         self._nfs = nfs
+        self._disk = diskfile
```

Comment:
```suggestion
        self._diskfile = diskfile
```

Reply:
I have corrected it. Thanks for pointing it out.

Reply:
Thanks.  Ah I see the GitHub checks didn't run because this is your first set of contributions.  It should have picked it up otherwise, running it now anyway.